fiovb/ta/entry.c

```diff
@@ -336,11 +336,10 @@ static TEE_Result delete_persist_value(uint32_t pt,
 	name_buf_sz = params[0].memref.size;
 
 	/*
-	 * Vendor variables and rollback_protection should not be
+	 * rollback_protection should not be
```

Comment:
nit, mind having this comment in one single line now that is shorter?

Reply:
done